IdentityCore/src/oauth2/MSIDTokenResponse.m

```diff
@@ -101,7 +101,7 @@ - (MSIDAccountType)accountType
 
 - (MSIDErrorCode)oauthErrorCode
 {
-    return MSIDErrorCodeForOAuthError(self.error, MSIDErrorInteractionRequired);
+    return MSIDErrorCodeForOAuthError(self.error, MSIDErrorServerOauth);
```

Comment:
will the clients know to show UI on MSIDErrorServerOauth? I think the reason to have MSIDErrorInteractionRequired as default is so that client show UI in MSAL? Can we check what was MSAL and ADAL implementation before?

Reply:
Top level : Interaction required
Underlying : ServerOAuth

and double check converter in MSAL and ADAL